packages/workflow-apps-defaults/src/index.js

```diff
@@ -17,7 +17,7 @@ if (process.browser) {
         return {
           Terminal: require('workflow-app-iterm'),
           Browser: require('workflow-app-safari'),
-          TextEditor: require('workflow-app-atom'),
+          TextEditor: require('workflow-app-textedit'),
```

Comment:
This might be missing from the `package.json` of `workflow-apps-defaults` :)

Reply:
😅